starter-code/views/search-form.ejs

```diff
@@ -0,0 +1,8 @@
+<body>
+  <form action="search" method="POST">
+    <label for="joke">
+      <input id="joke" type="text" name="jokeKeyword">Search a joke</input>
+    </label>
+    <input type="submit" value"searchQuery">Search!</input>
```

Comment:
The `value` of an `input` of type `submit` decides of the text that appears in the button.
What you want is `<input type="submit" value="Search!">`.